cmd/fissh/main.go

```diff
@@ -65,6 +67,23 @@ func main() {
 	}
 }
 
+func extractTimezoneFromEnv(env []string) (string, error) {
+	const envName = "TZ="
+	idx := slices.IndexFunc(env, func(s string) bool { return strings.HasPrefix(s, envName) })
+	if idx == -1 {
+		return "", nil
+	}
+
+	timezone := env[idx][len(envName):]
+
+	_, err := time.LoadLocation(timezone)
+	if err != nil {
+		return "", err
+	}
+	return timezone, nil
+
+}
+
```

Comment:
Also, happy to move this to the timezone package if y'all want.

Reply:
that would be nice, but not a huge deal